### PR TITLE
homepage added

### PR DESCRIPTION
**Pull request by pariyar07**
Screenshot 2022-03-21 at 12 30 30 AM

### CODE REVIEW

File: src/components/Home/Main/index.jsx [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+                    <li className="feature-list">
+                        <img className="feature-img" src={require("./shoes.png")} alt="" />
+                        <span className="feature-badge">Nike Air Max 270</span>
+                    </li>
+                    <li className="feature-list">
+                        <img className="feature-img" src={require("./shoes.png")} alt="" />
+                        <span className="feature-badge">Nike VaporMax</span>
+                    </li>
+                    <li className="feature-list">
+                        <img className="feature-img" src={require("./shoes.png")} alt="" />
```
Comment (ivinayakg) [inline src/components/Home/Main/index.jsx]: Try not import your images like this, instead put them in the public folder and then access it.

**Review by ivinayakg — APPROVED** [review]

File: src/components/Home/Main/index.jsx (hunk 1 above)
Comment (pariyar07) [reply src/components/Home/Main/index.jsx]: Will do that thanks.

**Review by ravikumar0403 — APPROVED** [review]
code is clean, looks good

**Review by MaruthiPothuganti — COMMENTED** [review]
Code looks clean and good. No changes recommended.

File: src/components/Home/Main/index.jsx (hunk 1 above)
Comment (MaruthiPothuganti) [reply src/components/Home/Main/index.jsx]: Yeah. What @ivinayakg suggested is a better way to do it as you would need to add require and path everytime you render an image.

**Pull request merged**